src/main/java/com/laserfiche/repository/api/clients/impl/EntriesClientImpl.java

```diff
@@ -651,15 +656,20 @@ else if (rawResponse.getStatus() == 429)
                                         exception[0] = new RuntimeException(rawResponse.getStatusText());
```

Comment:
Just noticed this exception was missed to throw ApiException. Could you update this else exception to be something like `ApiException.create(rawResponse.getStatus(), headersMap, problemDetails, null);`

Reply:
ok in this case then guess i can just replace lines 634 - 656 with `exception[0] = ApiException.create(rawResponse.getStatus(), headersMap, problemDetails, null);` since it seems to be the same thing logically

Reply:
ok done